src/crh/bowe_hopwood/constraints.rs

```diff
@@ -96,14 +109,15 @@ where
     P: TEModelParameters,
     W: Window,
 {
+    type InputVar = Vec<UInt8<ConstraintF<P>>>;
```

Comment:
This can actually still be `[UInt8<ConstraintF<P>>]`, I believe; the associated type doesn't have to be `Sized`, right?

Reply:
Currently `InputVar` is sized. Let me modify the trait a little bit

Reply:
fixed